sdk/webpubsub/test-resources.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
I'll leave a pointer to https://github.com/Azure/azure-sdk-for-net/blob/master/eng/common/TestResources/README.md in case you want to try this before you merge.